fedimint-server/src/config/distributedgen.rs

```diff
@@ -506,6 +510,14 @@ pub trait PeerHandleOps {
         dkg_key: String,
         key: secp256k1::PublicKey,
     ) -> DkgResult<BTreeMap<PeerId, secp256k1::PublicKey>>;
+
+    async fn exchange_with_peers<T: Encodable + Decodable + Send + Sync>(
```

Comment:
Please document the semantics of the function. Does it send to everyone, what about itself?

Reply:
IMO docs should be on function declaration in trait, not on implementation because goto definition takes you to trait definition.

Reply:
good call, moved to the trait. Also added one for `exchange_pubkeys`